systemtests/src/main/java/io/enmasse/systemtest/messaginginfra/ResourceManager.java

```diff
@@ -229,13 +236,14 @@ public MqttClientFactory getMqttClientFactory() {
 
     @SafeVarargs
     public final <T extends HasMetadata> void deleteResource(T... resources) throws Exception {
+        LOGGER.info("Deleting {} resources", resources.length);
```

Comment:
@lulf I don't see any useful reason why logging count of resources helps me in debugging or reading logs

Reply:
Ok, I can remove it. But I will still want to keep the per-address logging at the debug level, because it is way to spammy (and I prefer logging lot of info if there is an error, not if there is not an error).

Reply:
per address? it is not only for address but for all resources, right? I don't know, we can discuss it, but I prefer to see creating resources on info level

Reply:
I'm not sure its useful to know if it is being created when everything is working, but I do see its useful when it fails. I think maybe what we should've had, is a log buffer that we keep for each test case. All logging happens to this buffer, and it is printed if test fails. If test succeeds, it is just cleared.

Wouldn't this simplify reading logs in event of errors a lot? 

Out of the scope of this PR obviously, so I'll revert the logging changes for now.

Reply:
This is makes sense, yeah it is out of scope for this PR, but I really love this idea